easybuild/easyblocks/t/tensorflow.py

```diff
@@ -0,0 +1,260 @@
+
+            gcc_lib_inc_fixed = os.path.join(os.path.dirname(gcc_lib_inc), 'include-fixed')
+            if not os.path.exists(gcc_lib_inc_fixed):
+                raise EasyBuildError("Derived directory %s does not exist", gcc_lib_inc_fixed)
+
+            gcc_cplusplus_inc = os.path.join(gcc_root, 'include', 'c++', gcc_ver)
+            if not os.path.exists(gcc_cplusplus_inc):
+                raise EasyBuildError("Derived directory %s does not exist", gcc_cplusplus_inc)
+        else:
+            raise EasyBuildError("Failed to determine installation prefix for GCC")
+
+        inc_paths = [gcc_lib_inc, gcc_lib_inc_fixed, gcc_cplusplus_inc]
+        lib_paths = [gcc_lib64]
+
+        cuda_root = get_software_root('CUDA')
+        if cuda_root:
+            inc_paths.append(os.path.join(cuda_root, 'include'))
+            lib_paths.append(os.path.join(cuda_root, 'lib64'))
+
+        regex_subs = [
+            (r'-B/usr/bin/', '-B%s/ %s' % (binutils_bin, ' '.join('-L%s/' % p for p in lib_paths))),
+            (r'(cxx_builtin_include_directory:).*', '\n  '.join(r'\1 "%s"' % resolve_path(p) for p in inc_paths)),
+        ]
+        for tool in ['ar', 'cpp', 'dwp', 'gcc', 'gcov', 'ld', 'nm', 'objcopy', 'objdump', 'strip']:
+            path = which(tool)
+            if path:
+                regex_subs.append((os.path.join('/usr', 'bin', tool), path))
+            else:
+                raise EasyBuildError("Failed to determine path to '%s'", tool)
+
+        if self.toolchain.options.get('pic', None):
+            # -fPIE/-pie and -fPIC are not compatible, so patch out hardcoded occurences of -fPIE & -pie
+            regex_subs.extend([('-fPIE', '-fPIC'), ('"-pie"', '"-fPIC"')])
+
+        for path, dirnames, filenames in os.walk(self.cfg['start_dir']):
+            for filename in filenames:
+                if filename.startswith('CROSSTOOL'):
+                    full_path = os.path.join(path, filename)
+                    self.log.info("Patching %s", full_path)
+                    apply_regex_substitutions(full_path, regex_subs)
+
+        tmpdir = tempfile.mkdtemp(suffix='-bazel-build')
+
+        cmd = ['bazel', '--output_base=%s' % tmpdir, 'build']
+        # https://docs.bazel.build/versions/master/user-manual.html#flag--compilation_mode
+        cmd.append('--compilation_mode=opt')
+        # https://docs.bazel.build/versions/master/user-manual.html#flag--config
+        cmd.append('--config=opt')
+        # https://docs.bazel.build/versions/master/user-manual.html#flag--subcommands
+        # https://docs.bazel.build/versions/master/user-manual.html#flag--verbose_failures
+        cmd.extend(['--subcommands', '--verbose_failures'])
+
+        if self.toolchain.options.get('pic', None):
+            cmd.append('--copt="-fPIC"')
+
+        cmd.append(self.cfg['buildopts'])
+
+        # pass through environment variables that may specify location of license file for Intel compilers
+        for key in []:
```

Comment:
I have the feeling that this shouldn't be an empty list....

Reply:
Woops, yes, I did the for for `$INTEL_LICENSE_FILE` through this initially, but it turns out that doesn't actually work...